src/relay/backend/te_compiler_cache.cc

```diff
@@ -193,7 +193,7 @@ class LowerToTECompute : public backend::MemoizedExprTranslator<Array<te::Tensor
     } else {
       const auto* ttype = op->checked_type().as<TensorTypeNode>();
       std::stringstream ss;
-      ss << "constant_" << const_index++;
+      ss << "cc_constant_" << const_index++;
```

Comment:
i discussed this with @Mousius a bit. his view is that we should adopt an intermediate solution to this problem to bugfix, and then take up the architectural issue later.

How about the following compromise: 
- here we fix the bug so we can release without it by using `candidate_name_` as part of the prefix so it's not just randomly `cc_constant_N`.
- I wrote up the broader architectural issue in a [Discuss pre-RFC](https://discuss.tvm.apache.org/t/pre-rfc-name-mangling-in-irmodules/12944) and would love comments there.